src/guides/v2.4/graphql/mutations/create-company-user.md

```diff
@@ -0,0 +1,179 @@
+---
+group: graphql
+title: createCompanyUser mutation
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+b2b_only: true
+---
+
+Use the `createCompanyUser` mutation to create a new company user or add a user from the existing customer for your company.
```

Comment:
Who can run this mutation? Anyone with a customer/company user token? Do they need to have the "Manage users and teams" role permission?

Rewrite the first paragraph to be something like thuis

The `createCompanyUser` mutation allows an existing company user with the <role name> role to create a new company user. The specified email address determines how Magento processes the request.

*  If the email address is unique for the website?/store?/store view?, Magento immediately creates the company user.
*  If the email address belongs to a customer who is not a company user, Magento sends an invitation to join the company organization to the customer. When the customer accepts the invitation, Magento adds the customer to the company organization.
*  If the email address belongs to an existing company user with the same company, Magento returns the error `A customer with the same email already assigned to company.`
*  If the email address belongs to a customer who is part of another company organization, ????